DESCRIPTION

```diff
@@ -14,9 +14,10 @@ Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.2.3
 Suggests: 
     testthat (>= 3.0.0),
-    Matrix
+    Matrix,
 Config/testthat/edition: 3
 Imports:
     methods,
-    Seurat,
+    Seurat (>= 5.0.0),
```

Comment:
In order to support v5 datasets, we need to bump the min version of Seurat.  Also, have a direct dependency on `SeuratObject` because I think using the `LayerData` method directly reads better.

Reply:
Could we keep the `Imports` on Seurat unversioned and `Suggests` >= 5.0.0? I am not sure if Roxygen will let you specify it that way, but I imagine cutting to Seurat v5 will trigger a lot of support tickets

Reply:
Agreed, making v5 a requirement is probably going to be an issue in the near term.

Reply:
will put in effort to not make this a requirement